package.json

```diff
@@ -44,13 +44,13 @@
     "eslint-loader": "4.0.2",
     "eslint-plugin-import": "2.26.0",
     "fast-glob": "3.2.2",
+    "gaze": "^1.1.3",
     "postcss": "8.4.6",
     "sass": "1.44.0",
     "stylelint": "14.6.1",
     "webpack": "4.46.0"
   },
   "devDependencies": {
-    "gaze": "^1.1.3",
```

Comment:
gaze is dev dependency used only locally for watcher, not needed in the pipeline (remote)

Reply:
Ah, I see the issue #65 , hm, how to address this?
I would not add devDependencies to dependencies list, because then it will be installed on every remote build, even PROD build in pipelines, where watch and other dev tasks are not needed.

Reply:
As agreen on slack, we'll move on with this change and @raohmaru will work on removing the gaze dependency altogether in a separate PR